src/renderer/components/ft-list-channel/ft-list-channel.js

```diff
@@ -53,11 +53,8 @@ export default defineComponent({
 
       this.channelName = this.data.name
       this.id = this.data.id
-      if (this.hideChannelSubscriptions || this.data.subscribers == null) {
-        this.subscriberCount = null
-      } else {
-        this.subscriberCount = this.data.subscribers.replace(/ subscriber(s)?/, '')
-      }
+      this.subscriberCount = this.data.subscribers != null ? this.data.subscribers.replace(/ subscriber(s)?/, '') : ''
```

Comment:
Probably best to keep this as `null` when there are no subscribers in the response, as that's what the Vue template uses to hide them from the UI. (see https://github.com/FreeTubeApp/FreeTube/pull/3692/files#diff-efe7580cfb6e0f2be7d806690c3c0460ed7c9d0b1fc7c72c4207824af09b6b73R29)
```suggestion
      this.subscriberCount = this.data.subscribers != null ? this.data.subscribers.replace(/ subscriber(s)?/, '') : null
```

Reply:
Good catch, thank you